go.mod

```diff
@@ -62,41 +62,41 @@ require (
 	github.com/prometheus/common v0.32.1
 	github.com/prometheus/prometheus v2.5.0+incompatible
 	github.com/ryanuber/go-glob v1.0.0
-	github.com/spf13/cobra v1.3.0
+	github.com/spf13/cobra v1.4.0
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.10.1
 	github.com/vishvananda/netlink v1.1.1-0.20210330154013-f5de75959ad5
 	github.com/yl2chen/cidranger v1.0.2
 	go.opencensus.io v0.23.0
 	go.opentelemetry.io/proto/otlp v0.12.0
 	go.uber.org/atomic v1.9.0
-	go.uber.org/multierr v1.7.0
-	golang.org/x/net v0.0.0-20220127200216-cd36cc0744dd
-	golang.org/x/oauth2 v0.0.0-20211104180415-d3ed0bb246c8
+	go.uber.org/multierr v1.8.0
+	golang.org/x/net v0.0.0-20220624214902-1bab6f366d9e
+	golang.org/x/oauth2 v0.0.0-20220309155454-6242fa91716a
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
-	golang.org/x/sys v0.0.0-20220128215802-99c3d69c2c27
+	golang.org/x/sys v0.0.0-20220627191245-f75cf1eec38b
 	golang.org/x/time v0.0.0-20211116232009-f0f3c7e86c11
 	gomodules.xyz/jsonpatch/v3 v3.0.1
-	google.golang.org/api v0.66.0
-	google.golang.org/genproto v0.0.0-20220202230416-2a053f022f0d
-	google.golang.org/grpc v1.44.0
-	google.golang.org/protobuf v1.27.2-0.20220217170731-3992ea83a23c
+	google.golang.org/api v0.73.0
+	google.golang.org/genproto v0.0.0-20220628213854-d9e0b6570c03
+	google.golang.org/grpc v1.47.0
+	google.golang.org/protobuf v1.28.0
 	gopkg.in/square/go-jose.v2 v2.6.0
 	gopkg.in/yaml.v2 v2.4.0
-	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
+	gopkg.in/yaml.v3 v3.0.1
 	helm.sh/helm/v3 v3.8.0
-	istio.io/api v0.0.0-20220718214027-666b55ffb5a8
-	istio.io/client-go v1.13.5-0.20220718214527-fcd2b1530308
-	istio.io/pkg v0.0.0-20220413180906-765b7512325f
-	k8s.io/api v0.23.3
+	istio.io/api v0.0.0-20220718195626-87dfc8c8c07a
```

Comment:
Something is wrong with the update_deps as it's pulling the incorrect version of the other repos. This commit is from the master branch of istio/api for example. It should be the latest from the 1.13 branch. I think there is a env var that the script will use to get from the appropriate branch. I suspect it's default is master.